fedgov-cv-ISDcreator-webapp/src/main/webapp/js/features.js

```diff
@@ -135,10 +140,181 @@ function buildDots(i, j, dot, latLon, lanes, laneMarkers){
 	}
 }
 
+
+function placeComputedLane(newDotFeature, lanes, vectors, laneMarkers, laneWidths, computingLane, computedLaneSource, speedForm, sharedWith, laneTypeOptions, typeAttributeNameSaved, controls) {
```

Comment:
can you add a description to this function definition

Reply:
Addressed :https://github.com/usdot-fhwa-stol/connectedvcs-tools/commit/c9ff7b20a2a576b1be6f30d01513057cbdcd1f43